tools/test_commit_range.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+###########################################################################
+#
+#  Copyright 2016, ARM Limited, All Rights Reserved
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+###########################################################################
+
+# Check if all commits since `origin/master` build
+
+COMMITS=$(git rev-list --reverse origin/master..HEAD)
```

Comment:
Could we pass `remote/reference` as a cmd line arg? It should be easy to add. In my case I never have `origin` for example, but the actual remote names, `ARMmbed`, `AlessandroA`, etc. No cmd line arg could default to `origin/master`